src/libical-glib/api/i-cal-unknowntokenhandling.xml

```diff
@@ -21,10 +21,10 @@
     </enum>
     <method name="i_cal_get_unknown_token_handling_setting" corresponds="ical_get_unknown_token_handling_setting" kind="get" since="1.0">
         <returns type="ICalUnknowntokenhandling" comment="The setting of #ICalUnknowntokenhandling"/>
-        <comment>Get the setting of #ICalUnknowntokenhandling</comment>
+        <comment>Gets the setting of #ICalUnknowntokenhandling</comment>
     </method>
     <method name="i_cal_set_unknown_token_handling_setting" corresponds="ical_set_unknown_token_handling_setting" kind="set" since="1.0">
         <parameter type="ICalUnknowntokenhandling" name="newSetting" comment="A #ICalUnknowntokenhandling"/>
-        <comment>Set the setting of #ICalUnknowntokenhandling</comment>
+        <comment>Sets the setting of #ICalUnknowntokenhandling</comment>
```

Comment:
"Set the setting .." is strange sounding.  How about simply "Sets #ICalUnknowntokenhandling"

Reply:
Good idea. Done.